shared/charts/istio-gke-helper/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+name: istio-gke-helper
```

Comment:
I would name it `istio-gke-pdb`, since it contains only PDB resources, but current name is fine if the plan is to expand its functionality in future.

Reply:
Yep, I didn't want to name is `-pdb` only, as we might want to add some more functionality later.